whelktool/src/main/groovy/whelk/datatool/WhelkTool.groovy

```diff
@@ -819,6 +832,15 @@ class WhelkTool {
         }
     }
 
+    private static ValidationMode parseValidationMode(String arg) {
```

Comment:
It should fail on an invalid option instead of silently defaulting

Reply:
[c1eb8ae](https://github.com/libris/librisxl/pull/1518/commits/c1eb8ae77a2197970d3c432b6a3371fbb0e6b9e5)